packages/types/src/clerk.ts

```diff
@@ -460,6 +466,11 @@ export interface ClerkOptions {
   /** Optional support email for display in authentication screens */
   supportEmail?: string;
   touchSession?: boolean;
+  /**
+   * @experimental
+   * Indicates if the app is a Satellite app.
+   */
+  isSatellite?: boolean;
```

Comment:
:upside_down_face: Let's remove the property descriptions until we finalize the API. Keep the "@experimental" annotation though. :+1: